### PR TITLE
Docker compose

### PR DESCRIPTION
**Pull request by PaulusParssinen**
It's docker composing time.

Review/test this PR by following [the new Contributing guide](https://github.com/Ilmastokompassi/Ilmastokompassi/blob/docker-compose/docs/CONTRIBUTING.md).

## TLDR
Do following
1. ``git pull && git switch docker-compose``
2. ``sed 's/dev-db/localhost/g' backend/.env.dev > backend/.env``
   - This copies the new configuration, replaces the ``dev-db`` hostname with ``localhost`` from the database uri
     -  The ``dev-db`` hostname and the [.env.dev](https://github.com/Ilmastokompassi/Ilmastokompassi/blob/docker-compose/backend/.env.dev) are only used inside the docker compose environment to access the postgres database. Each docker container has own IP assigned to themselves, so the ``localhost`` means different thing there. That's why we use ``dev-db`` to resolve our local postgres database IP address instead.
4. ``bin/dev``
   - In separate terminal, run: ``backend/bin/db-migrate.sh``
5. Test that your normal development loop works:
   1. ``(cd backend && poetry run invoke test)``
   2. ``(cd frontend && npm run test)``
   3. ``(cd frontend && npm run cypress:headless)``

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ This is a Software production project developed during University of Helsinki's
 
 ## Documents
 
-- [Definition of done](documentation/definition_of_done.md)
-- [Workflow guide](documentation/workflow_guide.md)
-- [Environment setup workflow guide](documentation/environment_setup_workflow.md)
-- [Work hours](documentation/work_hours)
+- [Definition of done](docs/definition_of_done.md)
+- [Workflow guide](docs/workflow_guide.md)
```
Comment (trickwide) [inline README.md:17]:
```suggestion
- [How to contribute](docs/CONTRIBUTING.md)
```

File: README.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ This is a Software production project developed during University of Helsinki's
...
+- [Environment setup workflow guide](docs/environment_setup_workflow.md)
```
Comment (trickwide) [inline README.md:18]:
```suggestion
```

**Review by trickwide — CHANGES_REQUESTED** [review]
Changes work on my machine, great job! Added few suggestions to `README.md` as you've deleted `environment_setup_workflow.md` and `workflow_guide.md` we don't need to link them in `README.md` and we should add link to `CONTRIBUTING.md`.

**Review by trickwide — APPROVED** [review]

**Review by Desipeli — APPROVED** [review]
LGTM Everything works on my machine

**Pull request merged**